docs/quickstart.rst

```diff
@@ -82,7 +82,8 @@ work-station 2 (the central computer, look for ``ws2`` in the title at the top o
 
 If you can't find the right environments, please ask the instrument scientist.  This is important to make sure
 that there is a clean start for your experiment.  However, if later in your experiment you ever have to restart
-your environments, then you type at the command line ``bsui`` for the collection and ``ianalysis`` for the analysis environments, respectively.
+your environments, then you type at the command line ``bsui`` for the collection environment.
```

Comment:
good, I like this behavior, and this description. Use of "environments" may be confusing to people.  It is it explained sufficiently above?